samples/assistant/README.md

```diff
@@ -0,0 +1,221 @@
+# Assistants
```

Comment:
There should also be a reference to Assistant README in root level where all extension features are listed.

Reply:
Ah, yes, I forgot to include that. Will add.